### PR TITLE
Fix skeleton files

### PR DESCRIPTION
**Pull request by bennett000**
This fixes the immediate problems from #266

### CODE REVIEW

**Comment by bennett000** [comment]
By immediate problems I mean there's another PR on deck that has more fixes for the git hook issue

File: src/api/0.1/project-fs/skeletons/git-post-commit [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (xorgy) [inline src/api/0.1/project-fs/skeletons/git-post-commit]: AFAIK the correct approach is `#!/bin/env bash`

File: src/api/0.1/project-fs/skeletons/git-post-merge [hunk 2]
```diff
@@ -1 +0,0 @@
-./git-post-commit
```
Comment (xorgy) [inline src/api/0.1/project-fs/skeletons/git-post-merge]: Ditto, `#!/bin/env bash` is what I tend to see preferred.

File: src/api/0.1/project-fs/skeletons/git-pre-commit [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (xorgy) [inline src/api/0.1/project-fs/skeletons/git-pre-commit]: you know the drill.

**Comment by xorgy** [comment]
otherwise :shipit:

**Pull request merged**